articles/search/cognitive-search-how-to-debug-skillset.md

```diff
@@ -153,15 +153,35 @@ If skills produce output but the search index is empty, check the field mappings
 
 Custom skills can be more challenging to debug because the code runs externally, so the debug session can't be used to debug them. This section describes how to locally debug your Custom Web API skill, debug session, Visual Studio Code and [ngrok](https://ngrok.com/docs). This technique works with custom skills that execute in [Azure Functions](../azure-functions/functions-overview.md) or any other Web Framework that runs locally (for example, [FastAPI](https://fastapi.tiangolo.com/)).
 
-### Run ngrok
+### Get a Public URL
 
-[**ngrok**](https://ngrok.com/docs) is a cross-platform application that can create a tunneling or forwarding URL, so that internet requests reach your local machine. Use ngrok to forward requests from an enrichment pipeline in your search service to your machine to allow local debugging.
+#### Using Tunnelmole
+Tunnelmole is an open source tunneling tool that can create a Public URL that forwards to your local machine through a tunnel.
```

Comment:
line 159, "forwards" is missing an object.  How about "forwards requests to your local host through a tunnel"